src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +14,20 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        StringBuilder words = new StringBuilder();
+        try (Scanner scanner = new Scanner(file)) {
+            while (scanner.hasNext()) {
+                String word = scanner.next();
+                if (word.startsWith("w") || word.startsWith("W")) {
+                    words.append(word.toLowerCase().replaceAll("[^a-z]","")).append(" ");
+                }
+            }
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```

Comment:
From the checklist: Don’t ignore exceptions. Re-throw them as RuntimeException(e.g. throw new RuntimeException(e))